electricpy/__init__.py

```diff
@@ -4465,10 +4470,59 @@ def tap_changing_transformer(v1, v2, p, q, r, x):
         
     '''
 
-    ts = (v2 / v1) * (1 / (1 - (r * p + x * q) / (v1 * v2)))
+    ts = (Vgen*Vgen) / (Vgen*Vdis - (R * Pload + X * Qload) )
 
     return pow(ts, 0.5)
 
+def suspension_insulators(number_capacitors, capacitance_ratio, Voltage):
+    '''
+        This function calculates the voltage across each capacitor in a suspension insulator strain
```

Comment:
This function is very exciting! Do we have a formula that we can illustrate for what's being calculated? I looked at the reference and couldn't find one, but that may have been my own failure.

Reply:
\sum_{i=1}^{n-2}Vi+Vn-1*(1+m)-Vn*M = 0 and  \sum_{i=1}^{n}Vi = V(transmission line) these two are the equation through which we can construct a matrix and solve for Vi (i in range(1,n)

Reply:
![20210117_221203.jpg](https://user-images.githubusercontent.com/52835045/104849699-29785480-5911-11eb-98dd-2a7abc6d2171.jpg)

Reply:
How's this for a render to place in the documentation?

![SuspensionInuslator](https://user-images.githubusercontent.com/33275230/104854778-86e2b600-58bd-11eb-9bdb-410906f05765.png)

Reply:
> \sum_{i=1}^{n-2}Vi+Vn-1*(1+m)-Vn*M = 0 and \sum_{i=1}^{n}Vi = V(transmission line) these two are the equation through which we can construct a matrix and solve for Vi (i in range(1,n)

I think that would be pretty good to add! Let's throw that into the docstring as follows (that way Sphinx will render the math nicely):
```
To perform the calculations described here, the following formulas are satisfied, and used
to construct a matrix used to solve for :math:`V_i \\text{i in range(1,n)}`.

.. math:: \\sum_{i=1}^{n-2} V_{i} + V_{n-1} \\cdot (1+m) - V_{n} \\cdot M = 0

.. math:: \\sum_{i=1}^{n} V_{i} = V_{\\text{transmission line}}
```

Would you mind sticking this into the docstring?

Also, the example you showed in that image seems very helpful. If you want to take a crack at making some LaTeX for it, I'd be very happy to see it included. Not necessary, but certainly would be helpful!

Reply:
hello sir what I found was dividing  equations 1,2,3 by m on both side increase the accuracy in matrix multiplication , so sir can I do that or  can we proceed with this formula

Reply:
Yeah! this looks good. I was just thinking that if you wanted to try putting the additional content from your example into the docs. This is great though!